### DISCUSSION TITLE
Error when I try to send a mail

### DISCUSSION THREAD

**Discussion by tic2024-big** [gophish/gophish#3283] [Q&A]
Hi, I did a complete sending profile and then I started a campaign to try if I could send e-mails but when I click on send it returns me the following error:

Max connection attempts exceeded - 535 5.7.8 Username and Password not accepted. For more information, go to 5.7.8 https://support.google.com/mail/?p=BadCredentials 5b1f17b1804b1-434aa7e5b9csm86819395e9.43 - gsmtp

I put my Gophish user and password and on the host smtp.gmail.com:587 because I want to send e-mails to Gmail addresses. I found answers in some sites that say the solution is to create an application password with Google but that option does not exist now, so I can't do that.

**Comment by adiogo42** [comment]
solution:
a) from the gophish server try to reach SMTP server and also validate if the SMTP port is open 
b) try to connect in other session with mail user defined to send message if it's work
